src/Native/Unix/Common/pal_utilities.h

```diff
@@ -16,6 +16,19 @@
 #include <type_traits>
 #include <unistd.h>
 
+#ifdef DEBUG
+#define assert_msg(cond, msg, errno) do \
+{ \
+  if(!(cond)) \
+  { \
+    fprintf(stderr, "%s (%d): error %d: %s. (%s)\n", __FILE__, __LINE__, errno, msg, strerror(errno)); \
```

Comment:
This matches the format MSBuild spots in build logs. The way we run xunit right now, that doesn't do anything, but it's as good a choice as any.